libs/frontend/feature-sites/src/lib/use-sites.ts

```diff
@@ -56,15 +56,16 @@ export interface ISitesFeature {
   validateSiteName: (name: string) => string | undefined
 }
 
+const loading = ref(false)
+const sites = ref<ISiteViewModel[]>()
```

Comment:
This change is for #252.
I changed some other places as well so that the `<CreateAssetModal>` in `Build.vue` will be the only place calling the `listSites` endpoint when the builder is mounted.
Since Build Page and other pages (Asset Manager, Site Dashboard) will never show up at the same time, I made these two properties into global refs for simplicity.